aea/manager/manager.py

```diff
@@ -198,6 +200,7 @@ def __init__(
             )
         self._started_event = threading.Event()
         self._mode = mode
+        self._password = password
```

Comment:
In mam we store it in memory for the duration of mam usage. On AEA Manager app say, we can ask for password at login.